backend/src/ml_space_lambda/authorizer/lambda_function.py

```diff
@@ -283,8 +285,15 @@ def lambda_handler(event, context):
             ) and Permission.ADMIN in user.permissions:
                 policy_statement["Effect"] = "Allow"
             elif requested_resource == "/project" and request_method == "POST":
-                # Anyone can create a project
-                policy_statement["Effect"] = "Allow"
+                # Get the latest app config
+                app_config = AppConfigurationModel.from_dict(
+                    app_configuration_dao.get(configScope="global", num_versions=1)[0]
+                )
```

Comment:
We're going to wind up needing to do this in quite a few places in the authorizer. Can we have a central location to call that we can cache this even if it's just a very short lived? Just something with a 10 second cachetools ttl decorator or something would probably be enough to make a big difference.

Reply:
Good idea, looking into the cachetools decorator.

Reply:
Took me a minute to figure out how to unit test it properly, but I've got it working, will push on next revision.